account-kit/react/src/components/auth/card/index.tsx

```diff
@@ -1,54 +1,55 @@
-import {
-  useCallback,
-  useLayoutEffect,
-  useMemo,
-  useRef,
-  type ReactNode,
-} from "react";
+import { useCallback, useLayoutEffect, useMemo, useRef } from "react";
 import { useSignerStatus } from "../../../hooks/useSignerStatus.js";
 import { IS_SIGNUP_QP } from "../../constants.js";
 import { useAuthContext } from "../context.js";
-import type { AuthIllustrationStyle, AuthType } from "../types.js";
 import { Step } from "./steps.js";
 import { Notification } from "../../notification.js";
 import { useAuthError } from "../../../hooks/useAuthError.js";
 import { Navigation } from "../../navigation.js";
 import { useAuthModal } from "../../../hooks/useAuthModal.js";
 import { useElementHeight } from "../../../hooks/useElementHeight.js";
+import { MissingUiConfigComponentError } from "../../../errors.js";
 
 export type AuthCardProps = {
-  hideError?: boolean;
-  header?: ReactNode;
-  showSignInText?: boolean;
-  illustrationStyle?: AuthIllustrationStyle;
-  // Each section can contain multiple auth types which will be grouped together
-  // and separated by an OR divider
-  sections?: AuthType[][];
   className?: string;
-  onAuthSuccess?: () => void;
 };
 
 /**
  * React component containing an Auth view with configured auth methods
+ * and options based on the config passed to the AlchemyAccountProvider
  *
  * @param props Card Props
- * @param props.header optional header for the card (good place to put your app name or logo)
- * @param props.showSignInText optional boolean to show the sign in text (defaults to true)
- * @param props.sections array of sections, each containing an array of auth types
+ * @param props.className Optional class name to apply to the card
  * @returns a react component containing the AuthCard
```

Comment:
quick nit, can you format the `@param` tags to look like `@param {Type} {name} {description}`. This will make it line up with the doc gen cli expectations

Reply:
bump